jujugui/static/gui/src/app/d3-components/d3-components.js

```diff
@@ -162,25 +162,25 @@ YUI.add('d3-components', function(Y) {
     _normalizeHandler: function(handler, module, selector) {
       var result = {};
 
-      if (L.isString(handler)) {
+      if (typeof(handler) === 'string') {
         result.callback = module[handler];
         result.phase = 'on';
       }
 
-      if (L.isObject(handler)) {
+      if (typeof(handler) === 'object' && handler !== null) {
```

Comment:
`!Array.isArray(handler)` I guess?

Can we create these as functions like YUI did? `utils.isString` `utils.isObject` etc. That also checks `!== null`?